manifests/fpm/pool.pp

```diff
@@ -168,6 +173,12 @@
     warning('You must include the php base class before using any php defined resources')
   }
 
+  if ($php::globals::php_version != undef) {
+    $php_version_major = regsubst($php::globals::php_version, '^(\d+)\.(\d+)$','\1')
```

Comment:
shouldn't the calculation of the major php version be done in the params or in a more global location?